### PR TITLE
fix to pdf plots - y-axis was displayed incorrectly

### PR DESCRIPTION
**Pull request by Kucharssim**
Fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/1488

### CODE REVIEW

File: R/commonDiscoverDistributions.R [hunk 1]
```diff
@@ -962,6 +962,7 @@
   }
 
   lineData <- ggplot2::layer_data(plot, 1)
+  lineData <- lineData[!is.infinite(lineData$y),]
```
Comment (vandenman) [inline R/commonDiscoverDistributions.R:965]: if you also want to exclude NA and NaN you could use `is.finite`.
``` r
u <- c(NA, NaN, -Inf, Inf)
names(u) <- u
!is.infinite(u)
#>  <NA>   NaN  -Inf   Inf 
#>  TRUE  TRUE FALSE FALSE
is.finite(u)
#>  <NA>   NaN  -Inf   Inf 
#> FALSE FALSE FALSE FALSE
```

**Review by vandenman — CHANGES_REQUESTED** [review]
Most test figures actually improved (the qq-plots now consistently show a line at x = y) but one figure is missing a line:

image

the grey line is gone in the added plot. Maybe the endpoints of that line also needs some adjustments?

File: R/commonDiscoverDistributions.R (hunk 1 above)
Comment (Kucharssim) [reply R/commonDiscoverDistributions.R:965]: Thanks!

**Comment by Kucharssim** [comment]
> the grey line is gone in the added plot.

So what I see here is that the grey line is *added* in the added plot. Or maybe I am looking at a different part?

Btw. the changes in the test plots have nothing to do with the fix, but with an update to jaspGraphs. I did not notice in https://github.com/jasp-stats/jaspDistributions/pull/25 that some of these plots missed these lines. After updating jaspGraphs on my computer, the figures started to fail, so I checked what's the problem to find out that the update fixed this issue that I was not aware about.

**Comment by vandenman** [comment]
> Or maybe I am looking at a different part?

Nope I confused what left and right was for a second.

> Btw. the changes in the test plots have nothing to do with the fix, but with an update to jaspGraphs.

That is unfortunate. Do you happen to know which commit by any chance?

**Review by vandenman — APPROVED** [review]

**Comment by Kucharssim** [comment]
> > Btw. the changes in the test plots have nothing to do with the fix, but with an update to jaspGraphs.
> 
> That is unfortunate. Do you happen to know which commit by any chance?

I don't know the exact commit that fixes it. What I do know that https://github.com/jasp-stats/jaspGraphs/commit/61c67a70ffb52d324b4a87455f6ecb15d6d73259 was still incorrect whereas https://github.com/jasp-stats/jaspGraphs/commit/769e48040c944e2c6ff549e2d9aeaf80da9951a2 is correct. Not sure whether it narrows it down enough for you... If you need to know, I can go on a little hunt to check which commit exactly fixed this.

**Comment by vandenman** [comment]
Nah it's okay for now. In the future, I would like to have some checks or tests to avoid downstream unit test failures, but that can wait a little longer.

**Pull request merged**